arrows/vtk/applets/color_mesh.cxx

```diff
@@ -398,6 +432,42 @@ class color_mesh::priv : public kwiver::arrows::vtk::mesh_coloration
       writer->Write();
       return true;
     }
+    else if (ext == ".las")
+    {
+      auto lgcs = vital::local_geo_cs();
+      read_local_geo_cs_from_file(lgcs, input_geo_origin_file_);
+
+      vtkSmartPointer<vtkPoints> inPts = mesh->GetPoints();
```

Comment:
suggestion: we may want to break out most of the rest of this scope into a helper function, maybe `save_mesh_las()` or similar.  That helper can go in this same file, it just helps to make the code more modular and readable.